.github/workflows/publishVSCode.yml

```diff
@@ -1,62 +1,59 @@
-name: VSCode Publish
-
+name: Publish in Microsoft Marketplace
 on:
-  workflow_call:
-    inputs:
-      version:
-        type: string
-        required: true
-    outputs:
-      result:
-        description: Set to published if the push to the VSCode marketplace was successful
-        value: ${{ jobs.publish.outputs.result }}
-      status: 
-        description: Set to complete if all tasks associated with the workflow were successful.
-        value: ${{ jobs.mergeDevelop.outputs.status }}
+  release:
+    types: [published]
 
-# There are two outputs that are a result of running this workflow
-# The steps.publsh.outputs.result is an indicator that the pulish to the vscode marketplace succeeded.
-# The steps.mergeDevelop.outputs.status is an indicator that all the publish steps completed.
-jobs: 
-  updating-sha-for-publish: 
+jobs:
+
+  publish-start-notification:
+    name: 'Publish Start Notification'
+    uses: ./.github/workflows/slackNotification.yml
+    secrets: inherit
+    with:
+      title: "Publish waiting for approval in github."
```

Comment:
Nit: single quotes for consistency

Reply:
uh...actually it looks like all these slack calls use double quotes though...I'll update the name for consistency